src/Command/ConsoleCommand.php

```diff
@@ -51,6 +51,8 @@ public function execute(Arguments $args, ConsoleIo $io)
             return static::CODE_ERROR;
         }
 
+        include APP . 'repl_functions.php';
```

Comment:
PHPStan says: :man_shrugging: 

> Constant APP not found.   

https://github.com/cakephp/app/runs/609209541

Reply:
instead of root I would probably put it into /src/Console/

Reply:
Some projects use `src/functions` folder to hold functions files. I kinda like that.

Reply:
> Constant APP not found.

Use `autoload_files` config of phpstan to load `tests/bootstrap.php` like here https://github.com/cakephp/cakephp/blob/c68b60be0198ecc534ef84d08f84007668ad4808/phpstan.neon#L9
You can also autoload `repl_functions.php` functions that way about modifying `composer.json`.